test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.50
+2.51
```

Comment:
we just need to do some change in the folder

> run_if_changed: '^test\/images\/agnhost\/'

no need to bump the version, add something to the readme per example, instead of bumping the version